app/api/routes.py

```diff
@@ -171,9 +169,9 @@ def get_resources():
         ]
     except Exception as e:
         logger.error(e)
-        return standardize_response(None, [{"code": "bad-request"}], "bad request", 400)
+        return standardize_response(payload=None, status_code=400)
 
-    return standardize_response(resource_list, None, "ok")
+    return standardize_response(payload=dict(data=resource_list), status_code=200)
```

Comment:
Since the default for `status_code` is 200, I don't think we need that to be specified here